apps/project-renderer/src/components/app/index.js

```diff
@@ -78,22 +87,12 @@ const App = ( {
 		);
 
 		return (
-			window
-				.fetch(
-					`https://api.crowdsignal.com/v4/projects/${ projectCode }/form`,
-					{
-						method: 'POST',
-						body: form,
-					}
-				)
-				.then( ( res ) => {
-					if ( ! res.ok ) {
-						throw new Error( res.status );
-					}
-
-					return res.json();
-				} )
-				.then( ( json ) => {
+			submitProjectForm(
+				projectCode,
+				form,
+				preview ? { preview: true } : {}
```

Comment:
Couldn't we just pass `{ preview }` to the query param?
IMO, I think it'd make the code a bit cleaner avoiding the ternary if condition.
I'm assuming we're validating the value of `preview` on the backend to check whether it's true or false.

Reply:
Well... I do agree the code would look better. Then again, `preview` is either `true` or `undefined` and when the JS-Request-PHP translation happens, sending a `{ preview }` as querystring ends up, at PHP's side, as a `(string) "undefined"` (which is not a *falsy* value).

So, I guess it's just a matter of where do we put the ugly code :)